cmake/Modules/GeneratePkgConfig.cmake

```diff
@@ -28,28 +28,35 @@ function(_compile_features_to_gcc_flags _res _features)
 endfunction()
 
 function(_get_target_property_merging_configs _var_name _target_name _propert_name)
-	get_target_property(vals ${_target_name} ${_propert_name})
+	get_property(vals TARGET ${_target_name} PROPERTY ${_propert_name})
 	if (NOT vals)
-		set(vals "")
```

Comment:
Unneeded because `get_property()` returns empty strings if property does not exist (unlike `get_target_property()`, which returns "-NOTFOUND").

Reply:
Empty string is what we needed, so its ok

Reply:
Perhaps a cleaner way would be to use `get_property(vals TARGET ${_target_name} PROPERTY ${_propert_name} SET)` to test if there is a value?

Reply:
You are right, `SET` is ideal in this case, because all other code dependent on this `vals` variable is closed in `if` statements.